instrumentation/src/main/scala/com/evolutiongaming/util/dispatchers/Instrumented.scala

```diff
@@ -35,10 +36,12 @@ object Instrumented {
       result
     }
 
-    apply(instruments)
+    val surround = BlockingTracker(config.blockingTracker)
+
+    apply(instruments, surround)
   }
 
-  def apply(instruments: List[Instrument]): Instrumented = {
+  def apply(instruments: List[Instrument], surround: Surround): Instrumented = {
```

Comment:
why this cannot be done in therms for `Instrument` element ?

Reply:
Unfortunately, `Instrumented` element isn't able to surround `BlockContext.withBlockingContext`, there is no means to set block context outside of `withBlockingContext` method.